packages/roosterjs-content-model-core/lib/corePlugin/selection/SelectionPlugin.ts

```diff
@@ -359,24 +367,42 @@ class SelectionPlugin implements PluginWithState<SelectionPluginState> {
                     }
 
                     if (collapsed && td) {
-                        const { node, offset } = normalizePos(
+                        this.setRangeSelectionInTable(
                             td,
-                            key == Up ? td.childNodes.length : 0
-                        );
-                        const range = this.editor.getDocument().createRange();
-
-                        range.setStart(node, offset);
-                        range.collapse(true /*toStart*/);
-
-                        this.setDOMSelection(
-                            {
-                                type: 'range',
-                                range,
-                                isReverted: false,
-                            },
-                            null /*tableSelection*/
+                            key == Up ? td.childNodes.length : 0,
+                            this.editor
                         );
                     }
+                } else if (key == 'TabLeft' || key == 'TabRight') {
+                    const revrse = key == 'TabLeft';
```

Comment:
nit: just small typo

Reply:
Good question, It works the same both ways. In an HTML Table, the order of the cells is the same in RTL than LTR. This tab change works on a 2D representation of the HTML table where the cursor is.